src/Site_Command.php

```diff
@@ -520,7 +520,10 @@ private function get_network( $network_id ) {
 	 * 'url' isn't an available filter, as it comes from 'home' in wp_options.
 	 *
 	 * [--site__in=<value>]
-	 * : Only list the sites with these blog_id values (comma-separated).
+	 * : Only list 1 the sites with these blog_id values (comma-separated).
+	 *
+	 * [--site__user_in=<value>]
```

Comment:
I don't love this argument name, but I don't have a great alternative to suggest yet. Maybe `--user__in=<value>`?

Reply:
I was on the fence with that as well. It _feels_ like `--user_in` would be a better name. I'll update the whole code to that.